lightning/src/ln/channelmanager.rs

```diff
@@ -5321,8 +5321,8 @@ where
 			inbound_payment.expiry_time > header.time as u64
 		});
 
-		let mut pending_events = self.pending_events.lock().unwrap();
 		let mut outbounds = self.pending_outbound_payments.lock().unwrap();
+		let mut pending_events = self.pending_events.lock().unwrap();
```

Comment:
Should we document this lockorder dependency, and below?

Reply:
:man_shrugging: lockorder is now self-documenting cause tests fail :) As long as the codepath is hit, and the channelmanager later serialized, lockorder is pretty rigorously checked.